lib/kitchen/driver/azurerm.rb

```diff
@@ -20,6 +20,8 @@ class Azurerm < Kitchen::Driver::Base
       attr_accessor :resource_management_client
       attr_accessor :network_management_client
 
+      kitchen_driver_api_version 2
```

Comment:
I mainly added this cause I saw across the board the primary drivers have this set. Willing to pull if this is a big deal.